arch/risc-v/src/mpfs/mpfs_cache.c

```diff
@@ -159,7 +159,7 @@ void mpfs_enable_cache(void)
       for (inc = 0; inc < (MPFS_WAY_BYTE_LENGTH /
            MPFS_CACHE_BLOCK_BYTE_LENGTH); ++inc)
         {
-          *p_scratchpad = g_init_marker + inc;
+          *p_scratchpad = g_init_marker /*+ inc*/;
```

Comment:
What is the purpose of leaving it commented vs deleting?

Reply:
Sorry, my bad. Deleted.